src/scripts/main.js

```diff
@@ -1,3 +1,109 @@
 'use strict';
 
-// write code here
+const TBODY = document.querySelector('.field tbody');
+const ROWS = TBODY.rows;
+
+const APPEND_ROW_BUTTON = document.querySelector('.append-row');
+const REMOVE_ROW_BUTTON = document.querySelector('.remove-row');
+const APPEND_COLUMN_BUTTON = document.querySelector('.append-column');
+const REMOVE_COLUMN_BUTTON = document.querySelector('.remove-column');
+
+const getTableSize = () => {
+  const rows = TBODY.rows;
+  const cells = rows[0].cells;
+
+  return {
+    rows: rows.length,
+    cells: cells.length,
+  };
+};
+
+const createRow = (cellsSize, rowSize) => {
+  // Next line for test, disabled button works correctly
+  if (rowSize === 10) {
+    return;
+  }
+
+  if (rowSize === 9) {
+    APPEND_ROW_BUTTON.disabled = true;
+  }
+
+  if (rowSize === 2) {
+    REMOVE_ROW_BUTTON.disabled = false;
+  }
+
+  const row = document.createElement('tr');
+
+  for (let i = 0; i < cellsSize; i++) {
+    row.insertCell();
+  }
+
+  TBODY.append(row);
+};
+const removeRow = (rowSize) => {
+  if (rowSize === 3) {
+    REMOVE_ROW_BUTTON.disabled = true;
+  }
+
+  if (rowSize === 10) {
+    APPEND_ROW_BUTTON.disabled = false;
+  }
+  TBODY.deleteRow(rowSize - 1);
+};
+
+const createColumn = (cellsSize, rowSize) => {
+  // Next line for test, disabled button works correctly
+  if (cellsSize === 10) {
+    return;
+  }
+
+  if (cellsSize === 9) {
+    APPEND_COLUMN_BUTTON.disabled = true;
+  }
+
+  if (cellsSize === 2) {
+    REMOVE_COLUMN_BUTTON.disabled = false;
+  }
+
+  for (let i = 0; i < rowSize; i++) {
+    ROWS[i].insertCell();
+  }
+};
+
+const removeColumn = (cellsSize) => {
```

Comment:
Code quality issue: Similar to the other functions, the `removeColumn` function is also doing too many things. It would be better to separate these responsibilities into different functions.